lib/main.coffee

```diff
@@ -77,9 +88,10 @@ module.exports =
             '--output-format=text'
           ]
           if @rcFile
-            args.push "--rcfile=#{@rcFile}"
+            rcFile = @rcFile.replace(/%p/g, projDir).replace(/%f/g, path.dirname(file))
```

Comment:
I was using regex earlier as that's what was already being used but then changed it in a different commit. In light of @SpainTrain's comment I switched everything back to regex for the sake of consistency and flexibility.